manifests/concourse-manifest/github_auth/dev_ci_additional_users.yml

```
@@ -11,6 +11,11 @@ instance_groups:
               github:
                 users:
                   - (( append ))
-                  - kr8n3r
                   - poveyd
                   - barsutka
+                  - szd55gds
+                  - philandstuff
+                  - adityapahuja
+                  - Krenair
+                  - chrisfarms
+                  - blairboy362
```

```suggestion
                  - blairboy362
                  - kr8n3r
```

We don't want to remove Jani

---

My brain saw "kr8n3r" == "Krenair" :joy: 

...fixing